pwa/src/components/applications/applicationForm.tsx

```diff
@@ -42,10 +43,10 @@ export const ApplicationForm: React.FC<ApplicationFormProps> = ({ id }) => {
 
   React.useEffect(() => {
     setHeader({
-      title: "Applications",
+      title: `Applications ${application ? application.name : ""}`,
```

Comment:
This should be singular (we're currently in a single **application** and not in applications).